Objects/Converters/ConverterRevit/ConverterRevitShared/PartialClasses/ConvertCurves.cs

```diff
@@ -74,7 +76,7 @@ public ApplicationObject DetailCurveToNative(DetailCurve speckleCurve)
       {
         revitCurve = Doc.Create.NewDetailCurve(Doc.ActiveView, baseCurve);
       }
-      catch (Exception)
+      catch (Autodesk.Revit.Exceptions.ApplicationException)
```

Comment:
If this fails at all, is the whole DetailCurveToNative() broken? Could it be creating an incorrect native object?